ocs_ci/ocs/cluster.py

```diff
@@ -500,6 +500,80 @@ def enable_health_monitor(self, sleep=5):
     def disable_health_monitor(self):
         self.health_monitor_enabled = False
 
+    def get_ceph_cluster_iops(self):
+        """
+        The function gets the IOPS from the ocs cluster
+
+        Returns:
+            Total IOPS in the cluster
+        """
+
+        ceph_status = self.get_ceph_status()
+        for item in ceph_status.split("\n"):
+            if 'client' in item:
+                iops = re.findall(r'\d+\.+\d+|\d\d*', item.strip())
+                read_iops = iops[2]
+                write_iops = iops[3]
```

Comment:
There are cases where we will not see Read or Write IOP's, please do check that in testing.

Reply:
Done